modules/apps/contacts/contacts-web/src/main/java/com/liferay/contacts/web/internal/portlet/ContactsCenterPortlet.java

```diff
@@ -712,6 +714,12 @@ public void updateSocialRequest(
 		}
 
 		writeJSON(actionRequest, actionResponse, jsonObject);
+
+		long userNotificationEventId = ParamUtil.getLong(
+			actionRequest, "userNotificationEventId");
+
+		_updateArchived(
+			themeDisplay.getUserId(), userNotificationEventId, true);
```

Comment:
Why the additional `archived` parameter? It is always set to `true`. Can you remove that param?

Also, can you check the value of `userNotificationEventId`, and only call `_updateArchived` if it is not 0?

Finally, shouldn`t this code be moved just before `jsonObject.put("success", Boolean.TRUE);`? As a user, I'd prefer for the notification not to be marked as read if the operation failed.

Reply:
@adolfopa The code was updated based on your feedback by @quanhuynhces . Thanks!